cardoor/assets/js/carrental/login.js

```diff
@@ -0,0 +1,80 @@
+function logout() {
```

Comment:
Could you add a comment about the decision of short JWT token lifetimes. As a stretch goal this would have to be somehow invalidated on the server side via blacklist